perfmetrics/scripts/load_tests/python/load_generator/load_generator.py

```diff
@@ -0,0 +1,357 @@
+        lg_const.PER_75: np.percentile(np_array, 75),
+        lg_const.PER_90: np.percentile(np_array, 90),
+        lg_const.PER_95: np.percentile(np_array, 95),
+        lg_const.PER_99: np.percentile(np_array, 99)
+    }
+
+  def _compute_default_post_test_metrics(self, observations):
+    """Computes default post load test metrics using observations.
+
+    Computes latency related metrics (percentiles, min, max and mean) of tasks
+    performed over the course of load test.
+    """
+    # Time stamps
+    start_time = observations[lg_const.START_TIME]
+    end_time = observations[lg_const.END_TIME]
+    actual_run_time = end_time - start_time
+
+    # Latency stats
+    latency_stats_names = [
+        lg_const.PRE_TASKS_LAT_STATS, lg_const.TASKS_LAT_STATS,
+        lg_const.POST_TASKS_LAT_STATS
+    ]
+    result_names = [
+        lg_const.PRE_TASKS_RESULTS, lg_const.TASKS_RESULTS,
+        lg_const.POST_TASKS_RESULTS
+    ]
+    latency_stats = {}
+    for stat_name, result_name in zip(latency_stats_names, result_names):
+      lat_pts = [result[3] - result[2] for result in observations[result_name]]
+      lat_pers = self._compute_percentiles(lat_pts)
+      latency_stats[stat_name] = lat_pers
+
+    metrics = {
+        lg_const.START_TIME: start_time,
+        lg_const.END_TIME: end_time,
+        lg_const.ACTUAL_RUN_TIME: actual_run_time,
+        lg_const.TASKS_COUNT: len(observations[lg_const.TASKS_RESULTS])
+    }
+    metrics.update(latency_stats)
+    return metrics
+
+  def _dump_metrics_into_json(self, metrics, output_file):
+    """Dumps given metrics as JSON file in UTF-8 format given output directory.
+    """
+    with open(output_file, 'w', encoding='utf-8') as f_p:
+      json.dump(metrics, f_p)
+
+  def _print_default_metrics(self, metrics):
+    """Prints given default metrics to console.
+    """
+    actual_run_time = metrics[lg_const.ACTUAL_RUN_TIME]
+    latency_stats_names = [
+        lg_const.PRE_TASKS_LAT_STATS, lg_const.TASKS_LAT_STATS,
+        lg_const.POST_TASKS_LAT_STATS
+    ]
+    task_type_names = ['Pre', 'Task', 'Post']
+
+    # Time metrics
+    print('\nTime: ')
+    print('\tStart time (epoch): ', metrics[lg_const.START_TIME])
+    print('\tEnd time (epoch): ', metrics[lg_const.END_TIME])
+    print('\tActual run time (in seconds): ', actual_run_time)
+
+    # Task related
+    print('\nTasks: ')
+    print('\tTasks count: ', metrics[lg_const.TASKS_COUNT])
+    print('\tTasks per sec: ', metrics[lg_const.TASKS_COUNT] / actual_run_time)
+
+    # Latency metrics
+    print('\nTasks latencies: ')
+    for task_type, latency_stat_name in zip(task_type_names,
+                                            latency_stats_names):
+      print('\t', task_type, ': ')
+      print('\t\tMin (in seconds): ', metrics[latency_stat_name][lg_const.MIN])
+      print('\t\tMean (in seconds): ', metrics[latency_stat_name][lg_const.MAX])
+      print('\t\t25th Percentile (in seconds): ',
```

Comment:
Why are we skipping PER_75 and PER_99?

Reply:
Usually, 75th is not required, so removed 75th and added 99th.